### DISCUSSION TITLE
AML sources

### DISCUSSION THREAD

**Discussion by irux** [ballerine-io/ballerine#94] [General]
Hello, do you have any kind of AML sources you want to connect to the project ? any of them without paying ? or what do you have in mind ?

**Comment by alonp99** [comment]
Hey @irux , we have AML checks using comply advantage in our cloud API and we intend to add free integrations to our open-source products soon, like open sanctions and watchman.

We build our roadmap around our community use cases, I would love to learn more about yours.

Are you on our slack or discord servers? Let’s continue the conversation there: [Discord](https://discord.gg/e2rQE4YygA), [Slack](https://join.slack.com/t/ballerine-oss/shared_invite/zt-1iu6otkok-OqBF3TrcpUmFd9oUjNs2iw)
